app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    temp = command.split(" ")
+    old_file = temp[1]
+    new_file = temp[2]
+    if old_file == new_file:
+        exit()
+    with open(old_file, "r") as old:
+        with open(new_file, "w") as new:
```

Comment:
you can write 7 and 8 in 1 line

Reply:
@oksana0410 is right, fix this please
![image](https://user-images.githubusercontent.com/103201010/220109352-76e1ec1c-dc99-4738-9926-4861ae90ee1f.png)

Reply:
also new and old is a bad names for variables, use more descritptive names for naming